pkg/kubelet/cm/topologymanager/policy_single_numa_node.go

```diff
@@ -53,12 +55,134 @@ func (p *singleNumaNodePolicy) canAdmitPodResult(hint *TopologyHint) lifecycle.P
 	}
 }
 
+// Get the narrowest hint aligned to one NUMA node
+// assumption: allResourcesHints are not empty, all hints have NUMAAffinity != nil,
+// all hints have exactly one NUMA node set in NUMANodeAffinity
+func (p *singleNumaNodePolicy) getHintMatch(allResourcesHints [][]TopologyHint) (bool, TopologyHint) {
+	// Sort all resources hints
+	for _, resource := range allResourcesHints {
+		sort.Slice(resource, func(i, j int) bool {
+			if resource[i].NUMANodeAffinity.GetBits()[0] < resource[j].NUMANodeAffinity.GetBits()[0] {
+				return true
+			}
+			return false
+		})
```

Comment:
Is there a reason not to use the standard `bitmask.IsNarrowerThan()` operator here?